### PR TITLE
interface_generator modified not to update pre-generated JSON

### PR DESCRIPTION
**Pull request by knorth55**
interface_generator modified not to update pre-generated JSON

#1143 
#1136

### CODE REVIEW

File: jsk_2016_01_baxter_apc/scripts/interface_generator_pick.py [hunk 1]
```diff
@@ -88,108 +78,127 @@ def select_item():
...
-this_dir = os.path.dirname(os.path.abspath(__file__))
-output_dir = os.path.join(this_dir,'../json')
-if not os.path.exists(output_dir):
-    os.mkdir(output_dir)
-os.chdir(output_dir)
-with open('pick_layout_'+version+'.json', 'w') as outfile:
-    json.dump(data, outfile, sort_keys=True, indent=4, separators=(',',': '))
-print('pick_layout_'+version+'.json generated at ../json')
-os.chdir(this_dir)
+class interface_generator_pick():
```
Comment (wkentaro) [inline jsk_2016_01_baxter_apc/scripts/interface_generator_pick.py]: should be camel case.

**Comment by knorth55** [comment]
@wkentaro modified into camel case

**Comment by wkentaro** [comment]
Please squash commits.

**Comment by knorth55** [comment]
squashed

**Pull request merged**